torch/csrc/jit/mobile/promoted_prim_ops.cpp

```diff
@@ -99,15 +100,15 @@ void toList(Stack& stack) {
 
   // Rebuild the output type using elem_ty_val and dim_val. Start
   // with the element type corresponding to elem_ty_val.
-  TypePtr out_ty;
+  at::TypePtr out_ty;
   if (elem_ty_val == 0) {
-    out_ty = IntType::get();
+    out_ty = at::IntType::get();
```

Comment:
Since these operators become part of the core interpreter, we should use DynamicType from now.
For example, `IntType::get()` should become `DynamicTypeTrait<IntType>::getBaseType()`. If that looks tedious feel free to write some helper functions.